src/system_database.ts

```diff
@@ -168,8 +162,9 @@ export interface SystemDatabase {
   upsertEventDispatchState(state: DBOSEventReceiverState): Promise<DBOSEventReceiverState>;
 
   // Workflow management
-  getWorkflows(input: GetWorkflowsInput): Promise<GetWorkflowsOutput>;
-  getQueuedWorkflows(input: GetQueuedWorkflowsInput): Promise<GetWorkflowsOutput>;
+  getWorkflows(input: GetWorkflowsInput): Promise<WorkflowStatusInternal[]>;
```

Comment:
I know it's an annoying thing but is it doable to change this to `listWorkflow` and `listQueuedWorkflows`? We can do it in a separate PR too.